src/backend/env/user.rs

```diff
@@ -415,6 +419,7 @@ impl User {
                     .cycles
                     .checked_sub(amount)
                     .ok_or("wrong negative delta amount")?;
+                self.credits_burned += amount;
```

Comment:
Are these credits actually burned? IIUC, `change_credits` is also called when doing various operations with credits. For example, calling the `transfer_credits` endpoint seems to call this function.

If that's the case, then it is easy to game this lottery by just transferring credits between two accounts, no?

IMO, the credits burned should come from the useful user activity such as publishing a post or reacting.

Reply:
Thanks Ulan, yeah I realized yesterday night that there exists a similar issue with reports: the credits are actually burned when the user got reported, but this is not what we want to reward by increasing chances of a lottery...

So I will need to change this to only count the credits on content creation + reactions and nothing else.

Reply:
Please check the new commit. We now count only credits spent on positive reactions and new posts. We don't count excess penalties, negative reactions and nothing else where credits might be spent (like invites, etc).

Reply:
PS: only the fees (actually burned credits) are counted on reactions. The part that effectively gets transferred to the rewarded user will not count.